sbt-mode-hydra.el

```diff
@@ -339,21 +346,54 @@ x - clean        - reset substring (-- -z) to empty string
           (t
            (message "Do not know how to run project %s." project)))))
 
+(defun sbt-hydra:previous-command-play-run (play-project)
+  (string-match (format "%s/run" play-project) (car sbt-hydra:hydra-previous-commands)))
+
+(defun sbt-hydra:read-from-history ()
+  (cond ((fboundp 'ivy-read)
+         (ivy-read "History: " sbt-hydra:hydra-previous-commands))
```

Comment:
Hmm, in that case isn't it better to provide a variable/customization that enables ivy.